src/app/core/services/label.service.ts

```diff
@@ -48,6 +48,11 @@ const REQUIRED_LABELS = {
 @Injectable({
   providedIn: 'root'
 })
+
+/**
+ * Responsible for retrieval and parsing of label data from the GitHub
+ * repository for the CATcher application.
```

Comment:
Do you think we should include other functionalities of this service (like the label sync between Github and CATcher) or would it be too verbose?😅 something like 'responsible for retrieval, parsing, and syncing of label data'?

Reply:
Yea sure that'll be fine!